sonar-batch/src/main/java/org/sonar/batch/scan/filesystem/ProjectFileSystemAdapter.java

```diff
@@ -50,20 +43,14 @@
 
   private final DefaultModuleFileSystem target;
   private final PathResolver pathResolver = new PathResolver();
-  private final MavenProject pom;
 
-  public ProjectFileSystemAdapter(DefaultModuleFileSystem target, Project project, @Nullable MavenProject pom) {
+  public ProjectFileSystemAdapter(DefaultModuleFileSystem target, Project project) {
     this.target = target;
-    this.pom = pom;
 
     // previously MavenProjectBuilder was responsible for creation of ProjectFileSystem
     project.setFileSystem(this);
```

Comment:
![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Method 'Project.setFileSystem(...)' is deprecated.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)